reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -516,9 +516,11 @@ public final HttpClient baseUrl(String baseUrl) {
 	}
 
 	/**
-	 * Specifies whether GZip compression is enabled.
+	 * Specifies whether compression (gzip, Brotli, and zstd) is enabled.
```

Comment:
Reactor Netty does not support `zstd` out of the box. One needs to add the header by oneself.

```suggestion
	 * Specifies whether compression (gzip and Brotli) is enabled.
```

Reply:
@violetagg Thanks for the feedback!

I wanted to let users who want to use zstd encoding aware of this method as its invocation is required to use zstd encoding.

Is it okay if I add another note like the following?

```
<p>Note: For zstd compression, {@literal Accept-Encoding: zstd} header needs to be added explicitly.
```

Reply:
ok

Reply:
@violetagg Thanks! I added the note in b91b30bb9adfe4c2557818a2bc13a886cd20ef23.